app/src/main/java/com/weatherxm/ui/components/DailyRewardsCardView.kt

```diff
@@ -74,13 +81,23 @@ open class DailyRewardsCardView : LinearLayout, KoinComponent {
             context.getString(R.string.wxm_amount, formatTokens(it.toBigDecimal()))
         } ?: EMPTY_VALUE
 
-        binding.boosts.setVisible(data.totalBoostReward != null)
-        binding.noActiveBoosts.setVisible(data.totalBoostReward == null)
+        if(data.totalBoostReward == null) {
+            binding.boostsIcon.setImageDrawable(
+                AppCompatResources.getDrawable(context, R.drawable.ic_error_hex_filled)
+            )
+            binding.boostsIcon.setColor(R.color.midGrey)
+            binding.boosts.setVisible(false)
+            binding.noActiveBoosts.setVisible(true)
+        } else {
+            binding.boostsIcon.setImageDrawable(
+                AppCompatResources.getDrawable(context, R.drawable.ic_checkmark_hex_filled)
+            )
+            binding.boostsIcon.setColor(R.color.network_stats_chart_primary)
+        }
         data.totalBoostReward?.let {
```

Comment:
I think there is something wrong here in the recycler view. Take a look at the screenshot. If you scroll until the first "No active boosts" transaction and then scroll to the top, there are cards with checkmark icon along with "No Active Boosts"
![Screenshot_20240327_151640](https://github.com/WeatherXM/wxm-android/assets/10021503/ffba8bbb-4b2d-4f1d-9b13-f222009fd25c)